CHANGELOG.asciidoc

```diff
@@ -3,6 +3,880 @@
+
+*Packetbeat*
+
+
+*Winlogbeat*
+
+- Add Powershell module. Support for event ID's: `400`, `403`, `600`, `800`, `4103`, `4014`, `4105`, `4106`. {issue}16262[16262] {pull}18526[18526]
+- Fix Powershell processing of downgraded engine events. {pull}18966[18966]
+- Fix unprefixed fields in `fields.yml` for Powershell module {issue}18984[18984]
+
+*Functionbeat*
+
+
+==== Bugfixes
+
+*Affecting all Beats*
+
+- Fix potential race condition in fingerprint processor. {pull}18738[18738]
+- Add better handling for Kubernetes Update and Delete watcher events. {pull}18882[18882]
+- Fix config reload metrics (`libbeat.config.module.start/stops/running`). {pull}19168[19168]
+- Fix metrics hints builder to avoid wrong container metadata usage when port is not exposed {pull}18979[18979]
+- Server-side TLS config now validates certificate and key are both specified {pull}19584[19584]
+
+*Auditbeat*
+
+- system/socket: Fix dataset using 100% CPU and becoming unresponsive in some scenarios. {pull}19033[19033] {pull}19764[19764]
+
+*Filebeat*
+
+- Fix Kubernetes Watcher goroutine leaks when input config is invalid and `input.reload` is enabled. {issue}18629[18629] {pull}18630[18630]
+- Okta module now sets the Elasticsearch `_id` field to the Okta UUID value contained in each system log to minimize the possibility of duplicating events. {pull}18953[18953]
+- Fix `netflow` module to support 7 bytepad for IPFIX template. {issue}18098[18098]
+- Fix improper nesting of session_issuer object in aws cloudtrail fileset. {issue}18894[18894] {pull}18915[18915]
+- Fix Cisco ASA ASA 3020** and 106023 messages {pull}17964[17964]
+- Add missing `default_field: false` to aws filesets fields.yml. {pull}19568[19568]
+- Fix memory leak in tcp and unix input sources. {pull}19459[19459]
+- Fix Cisco ASA dissect pattern for 313008 & 313009 messages. {pull}19149[19149]
+- Fix bug with empty filter values in system/service {pull}19812[19812]
+
+*Heartbeat*
+
+
+*Journalbeat*
+
+
+*Metricbeat*
+
+- Fix incorrect usage of hints builder when exposed port is a substring of the hint {pull}19052[19052]
+- Stop counterCache only when already started {pull}19103[19103]
+- Remove dedot for tag values in aws module. {issue}19112[19112] {pull}19221[19221]
+- Fix empty field name errors in the application pool metricset. {pull}19537[19537]
+- Fix mapping of service start type in the service metricset, windows module. {pull}19551[19551]
+- Fix config example in the perfmon configuration files. {pull}19539[19539]
+- Fix k8s scheduler compatibility issue. {pull}19699[19699]
+- Fix SQL module mapping NULL values as string {pull}18955[18955] {issue}18898[18898
+
+*Packetbeat*
+
+- Fix process monitoring when ipv6 is disabled under Linux. {issue}19941[19941] {pull}19945[19945]
+
+*Winlogbeat*
+
+
+*Functionbeat*
+
+
+==== Added
+
+*Affecting all Beats*
+
+- Add configuration for APM instrumentation and expose the tracer trough the Beat object. {pull}17938[17938]
```

Comment:
No need to include technical implementation details in user facing changelog. Maybe better: `- Add initial instrument of Beats with APM GO Agent`.

Reply:
changed